File: flashnetclient/src/main/java/com/spectralogic/escapepod/flashnetclient/responses/Group.java
package com.spectralogic.escapepod.flashnetclient.responses;

import org.simpleframework.xml.Attribute;

public class Group {
    @Attribute(required = false)
    private String GroupName;

    @Attribute(name = "GroupAge.DWD", required = false)
    private int GroupAge;

    public String getGroupName() {
        return GroupName;
    }

    public int getGroupAge() {
        return GroupAge;
    }
}

File: flashnetclient/src/main/kotlin/com/spectralogic/escapepod/flashnetclient/responses/Group.kt
package com.spectralogic.escapepod.flashnetclient.responses

import org.simpleframework.xml.Attribute

data class Group(
        @field:Attribute(name = "GroupName", required = false)
        @param:Attribute(name = "GroupName", required = false)
        val GroupName : String?,

        @field:Attribute(name = "GroupAge.DWD", required = false)
        @param:Attribute(name = "GroupAge.DWD", required = false)
        val GroupAge : Int?
)
